metadata/IDCards__1__metadata.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
@vivekjhauipath do you see any problem if I rename the file from `IdCards` to `IDCards` now that the tar files are already generated?

Reply:
Do, you plan to change modelName also?

Reply:
@irinel-birsan-uipath : Please get this tested, I hope 2 packages don't get created, if IdCards was already synced.

Reply:
@vivekjhauipath In the file, the model name will remain `IDCards`. Only the file name was a bit different.
This is a new model which never reached the `main` branch.